src/System.Runtime/tests/TimeZoneInfoTests.netstandard1.7.cs

```diff
@@ -140,13 +140,27 @@ public static void ConvertTimeFromToUtc_UnixOnly()
             // Kind and source types don't match
             VerifyConvertToUtcException<ArgumentException>(new DateTime(2007, 5, 3, 12, 8, 0, DateTimeKind.Local), london);
 
+            // Test the ambiguous date
```

Comment:
So this failed in CI as it just happened to run during one of these ambiguous time regions?  Fun :smile:  Out of curiosity, how likely was that to happen?